openfl/federated/data/sources/pytorch/local_image_folder.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from pathlib import Path
+from typing import Union
+
+from PIL import Image
+
+from openfl.federated.data.sources.pytorch.local_folder import LocalFolder
+
+# from openfl.federated.data.sources.pytorch.local_folder import LocalFolder
+
+
+class LocalImageFolder(LocalFolder):
```

Comment:
To illustrate the new abstractions, I suggest refactoring one of the existing data loaders to the new data set/data source abstractions. For example:
https://github.com/securefederatedai/openfl/blob/develop/openfl-workspace/torch/histology/src/dataloader.py#L60

WDYT?